atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -149,6 +151,25 @@ def set_media_types(self, extra, worker_metadatas):
                 extra['image']['media_types'] = json.loads(annotations['media-types'])
                 return
 
+    def set_manifest_list_info(self, extra, worker_metadatas):
+        manifest_list_digests = self.workflow.postbuild_results.get(PLUGIN_GROUP_MANIFESTS_KEY)
+        # group_manifests returns either a list of manifest digests of the manifest lists,
+        # or a list of the image_manifests
```

Comment:
By the way, can we fix this? I think it should always return a list of manifest list digests, and that this list should be empty it didn't create any manifest list (i.e. when group=False). That way the test on the following line can be much simpler: `if manifest_list_digests:`.